config/config.go

```diff
@@ -108,17 +132,19 @@ func GetOptionalParameters(config *Config, defaultPort string,
 		logger.Info("setting slam system to localization mode")
 	}
 
-	useLiveData, err := DetermineUseLiveData(logger, config.UseLiveData, config.Sensors)
-	if err != nil {
-		return "", 0, 0, false, false, false, err
+	useLiveData := false
+	err := errors.New("")
+	if !modularizationV2Enabled {
+		useLiveData, err = DetermineUseLiveData(logger, config.UseLiveData, config.Sensors)
+		if err != nil {
+			return "", 0, 0, false, false, false, err
+		}
```

Comment:
is there a better way to initialize err here. I get an error if I do not declare it before line 137